results-processor/wptreport.py

```diff
@@ -565,7 +565,7 @@ def create_test_run(report, run_id, labels_str, uploader, auth,
 def main():
     parser = argparse.ArgumentParser(
         description='Parse and transform JSON wptreport.')
-    parser.add_argument('report', metavar='REPORT', type=str,
+    parser.add_argument('report', metavar='REPORT', type=str, nargs='+',
```

Comment:
Is this guaranteed to have at least one report? maybe '*' is a safer choice?

Reply:
Yes at least one report is required. The CLI interface is used by humans, so this is not a "guarantee" but a "requirement".